graphql_api/tests/test_repository.py

```diff
@@ -1432,201 +1451,3 @@ def test_desc_flake_rate_ordering_on_test_results(self) -> None:
                 {"node": {"name": test.name, "flakeRate": 0.2}},
             ]
         }
-
-    def test_test_results_aggregates(self) -> None:
```

Comment:
Proposing do the nesting of `testResultsAggregates` fields as a breaking change since no frontend client currently using it. Moved this test over to `test_test_analytics.py`